test/repeat_test.go

```diff
@@ -0,0 +1,74 @@
+package test
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/filecoin-project/go-f3/adversary"
+	"github.com/filecoin-project/go-f3/sim"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRepeat(t *testing.T) {
+	t.Parallel()
+
+	// Repeated messages should not interfere with consensus. Hence, test with larger than 1/3 adversary power.
```

Comment:
> Repeated messages should not interfere with consensus

This is a bit vague to me. The adversary copycats other participants (all of them, so it thus equivocates). When it has >1/3 of power, it is _necessary_ that it sends messages with votes matching some that it has received, otherwise consensus would not be possible. 

We could say that equivocations should not interfere with consensus up to 1/3 power, but with >1/3 power we should be able to engineer (or fluke) a divergent decision.

To demonstrate that we could detect a divergent decision if there was one, could you add a test where the adversary has enough power to convince two separate quarters of the network to decide different values?